src/vivarium/framework/results/manager.py

```diff
@@ -59,6 +65,8 @@ def setup(self, builder: "Builder"):
 
         self.get_value = builder.value.get_value
 
+        self.set_default_stratifications(builder.configuration.stratification.default)
```

Comment:
It's more in line with our conventions to define this method like `self.set_default_stratifications(builder)` and parse the configuration inside the method. This also has the benefit of moving all the logic of setting the default stratification inside the method defined to do that rather than having some of it outside in the `setup` function.

Reply:
Will update